src/mapgen_functions.cpp

```diff
@@ -3168,13 +3168,8 @@ void mapgen_basement_junk(map *m, oter_id terrain_type, mapgendata dat, int turn
         if( one_in( 1600 ) ) {
             m->furn_set( p, furn_str_id( "f_gun_safe_el" ) );
             if( one_in( 2 ) ) {
```

Comment:
Why does this if loop still remain?

Before the change you will always get something, now there is 50% change no loot at all.

Reply:
It's there because I don't know much about C++. Its removed.